### PR TITLE
Replace fc::optional with std::optional

### PR DESCRIPTION
**Pull request by TimothyBanks**
## Change Description
This PR is to replace the use of fc::optional with std::optional.  Based on performance metrics, std::optional is the better alternative.

Jira:  https://blockone.atlassian.net/browse/EPE-17
Jira issue has links to benchmarks describing performance improvement of `std::optional` over `fc::optional`.

## Change Type
**Select ONE**
- [ ] Documentation

- [ ] Stability bug fix

- [x] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Comment by TimothyBanks** [comment]
I need to look at the buildkite failures.  They are showing more places where I need to migrate to `std::optional` yet everything builds successfully on my MacOS clang setup.

File: libraries/chain/authorization_manager.cpp [hunk 1]
```diff
@@ -318,12 +318,12 @@ namespace eosio { namespace chain {
       }
 
       try {
-         optional<permission_name> linked_permission = lookup_linked_permission(authorizer_account, scope, act_name);
-         if( !linked_permission )
+         std::optional<permission_name> linked_permission = lookup_linked_permission(authorizer_account, scope, act_name);
+         if( !linked_permission.has_value() )
```
Comment (larryk85) [inline libraries/chain/authorization_manager.cpp:322]: `std::optional<T>` has an overloaded `bool` operator, so there is not a need to change these lines.

Comment (TimothyBanks) [reply libraries/chain/authorization_manager.cpp:322]: Yeah, I think I tracked down the issue with this.  The compiler was complaining about this but it appears that maybe it had to do with some implicit conversions that was causing ambiguity.

**Comment by TimothyBanks** [comment]
Closing to recreate a new PR to see if it fixes the buildkite failures.

**Comment by TimothyBanks** [comment]
This PR should be ready for review now.

I'm not quite sure what boxes to check for this though.

**Comment by TimothyBanks** [comment]
This PR is ready for final review.

File: libraries/chain/block_log.cpp [hunk 2]
```diff
@@ -303,9 +303,9 @@ namespace eosio { namespace chain {
       uint64_t      last_block_position() const { return read_buffer<uint64_t>(data() + size() - sizeof(uint64_t)); }
       chain_id_type chain_id() const { return preamble.chain_id(); }
 
-      fc::optional<genesis_state> get_genesis_state() const {
-         return std::visit(overloaded{[](const chain_id_type&) { return fc::optional<genesis_state>{}; },
-                                      [](const genesis_state& state) { return fc::optional<genesis_state>{state}; }},
+      std::optional<genesis_state> get_genesis_state() const {
+         return std::visit(overloaded{[](const chain_id_type&) { return std::optional<genesis_state>(); },
+                                      [](const genesis_state& state) { return std::optional<genesis_state>(state); }},
```
Comment (jeffreyssmith2nd) [inline libraries/chain/block_log.cpp:308]: Any reason to move away from uniform initialization here?

Comment (TimothyBanks) [reply libraries/chain/block_log.cpp:308]: I ran across a couple of places that looked like using `std::make_optional` was appropriate and I got overzealous with that change.  This is a remnant of that, so I'll definitely roll this back.

File: libraries/chain/include/eosio/chain/trace.hpp [hunk 3]
```diff
@@ -40,29 +40,29 @@ namespace eosio { namespace chain {
...
-      fc::optional<block_id_type>                producer_block_id;
-      fc::optional<transaction_receipt_header>   receipt;
+      std::optional<block_id_type>                producer_block_id;
+      std::optional<transaction_receipt_header>   receipt;
       fc::microseconds                           elapsed;
       uint64_t                                   net_usage = 0;
       bool                                       scheduled = false;
       vector<action_trace>                       action_traces;
-      fc::optional<account_delta>                account_ram_delta;
+      std::optional<account_delta>                account_ram_delta;
```
Comment (larryk85) [inline libraries/chain/include/eosio/chain/trace.hpp:61]: formatting

File: libraries/chain/include/eosio/chain/trace.hpp [hunk 4]
```diff
@@ -40,29 +40,29 @@ namespace eosio { namespace chain {
...
 
       transaction_trace_ptr                      failed_dtrx_trace;
-      fc::optional<fc::exception>                except;
-      fc::optional<uint64_t>                     error_code;
+      std::optional<fc::exception>                except;
```
Comment (larryk85) [inline libraries/chain/include/eosio/chain/trace.hpp:64]: formatting

File: libraries/chain/include/eosio/chain/trace.hpp [hunk 5]
```diff
@@ -40,29 +40,29 @@ namespace eosio { namespace chain {
...
+      std::optional<uint64_t>                     error_code;
```
Comment (larryk85) [inline libraries/chain/include/eosio/chain/trace.hpp:65]: formatting

File: libraries/fc/src/network/url.cpp [hunk 6]
```diff
@@ -74,7 +74,7 @@ namespace fc
          opath                     _path;
          ostring                   _query;
          ovariant_object           _args;
-         fc::optional<uint16_t>    _port;
+         std::optional<uint16_t>    _port;
```
Comment (larryk85) [inline libraries/fc/src/network/url.cpp:77]: formatting

File: libraries/state_history/include/eosio/state_history/transaction_trace_cache.hpp [hunk 7]
```diff
@@ -12,7 +12,7 @@ using chain::transaction_id_type;
 
 struct transaction_trace_cache {
    std::map<transaction_id_type, augmented_transaction_trace> cached_traces;
-   fc::optional<augmented_transaction_trace>                  onblock_trace;
+   std::optional<augmented_transaction_trace>                  onblock_trace;
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/transaction_trace_cache.hpp:15]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 8]
```diff
@@ -105,11 +105,11 @@ struct get_blocks_ack_request_v0 {
 struct get_blocks_result_v0 {
    block_position               head;
    block_position               last_irreversible;
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:107]: formatting (lines 108-112)

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 9]
```diff
@@ -151,30 +151,30 @@ struct action {
 struct action_trace_v0 {
    fc::unsigned_int             action_ordinal         = {};
    fc::unsigned_int             creator_action_ordinal = {};
-   fc::optional<action_receipt> receipt                = {};
+   std::optional<action_receipt> receipt                = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:154]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 10]
```diff
@@ -151,30 +151,30 @@ struct action {
...
    uint64_t                     receiver               = {};
    action                       act                    = {};
    bool                         context_free           = {};
    int64_t                      elapsed                = {};
    std::string                  console                = {};
    std::vector<account_delta>   account_ram_deltas     = {};
-   fc::optional<std::string>    except                 = {};
-   fc::optional<uint64_t>       error_code             = {};
+   std::optional<std::string>    except                 = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:161]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 11]
```diff
@@ -151,30 +151,30 @@ struct action {
...
+   std::optional<uint64_t>       error_code             = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:162]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 12]
```diff
@@ -151,30 +151,30 @@ struct action {
...
 };
 
 struct action_trace_v1 {
    fc::unsigned_int             action_ordinal         = {};
    fc::unsigned_int             creator_action_ordinal = {};
-   fc::optional<action_receipt> receipt                = {};
+   std::optional<action_receipt> receipt                = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:168]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 13]
```diff
@@ -151,30 +151,30 @@ struct action {
...
    uint64_t                     receiver               = {};
    action                       act                    = {};
    bool                         context_free           = {};
    int64_t                      elapsed                = {};
    std::string                  console                = {};
    std::vector<account_delta>   account_ram_deltas     = {};
    std::vector<account_delta>   account_disk_deltas    = {};
-   fc::optional<std::string>    except                 = {};
-   fc::optional<uint64_t>       error_code             = {};
+   std::optional<std::string>    except                 = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:176]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 14]
```diff
@@ -151,30 +151,30 @@ struct action {
...
+   std::optional<uint64_t>       error_code             = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:177]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 15]
```diff
@@ -201,7 +201,7 @@ struct partial_transaction_v1 {
    uint8_t                                    max_cpu_usage_ms       = {};
    fc::unsigned_int                           delay_sec              = {};
    std::vector<eosio::chain::extensions_type> transaction_extensions = {};
-   fc::optional<prunable_data_type>           prunable_data          = {};
+   std::optional<prunable_data_type>           prunable_data          = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:204]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 16]
```diff
@@ -218,14 +218,14 @@ struct transaction_trace_v0 {
    uint64_t                    net_usage         = {};
    bool                        scheduled         = {};
    std::vector<action_trace>   action_traces     = {};
-   fc::optional<account_delta> account_ram_delta = {};
-   fc::optional<std::string>   except            = {};
-   fc::optional<uint64_t>      error_code        = {};
+   std::optional<account_delta> account_ram_delta = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:221]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 17]
```diff
@@ -218,14 +218,14 @@ struct transaction_trace_v0 {
...
+   std::optional<std::string>   except            = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:222]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 18]
```diff
@@ -218,14 +218,14 @@ struct transaction_trace_v0 {
...
+   std::optional<uint64_t>      error_code        = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:223]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 19]
```diff
@@ -218,14 +218,14 @@ struct transaction_trace_v0 {
...
 
-   // semantically, this should be optional<transaction_trace>;
+   // semantically, this should be std::optional<transaction_trace>;
    // it is represented as vector because optional cannot be used for incomplete type
    std::vector<transaction_trace_recurse> failed_dtrx_trace = {};
-   fc::optional<partial_transaction>      partial           = {};
+   std::optional<partial_transaction>      partial           = {};
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:228]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 20]
```diff
@@ -238,9 +238,9 @@ using optional_signed_block = std::variant<chain::signed_block_v0_ptr, chain::si
 struct get_blocks_result_v1 {
    block_position               head;
    block_position               last_irreversible;
-   fc::optional<block_position> this_block;
-   fc::optional<block_position> prev_block;
-   optional_signed_block        block; // packed as fc::optional<fc::static_variant<signed_block_v0, signed_block>>
+   std::optional<block_position> this_block;
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:241]: formatting

File: libraries/state_history/include/eosio/state_history/types.hpp [hunk 21]
```diff
@@ -238,9 +238,9 @@ using optional_signed_block = std::variant<chain::signed_block_v0_ptr, chain::si
...
+   std::optional<block_position> prev_block;
```
Comment (larryk85) [inline libraries/state_history/include/eosio/state_history/types.hpp:242]: formatting

File: plugins/chain_plugin/chain_plugin.cpp [hunk 22]
```diff
@@ -148,15 +148,14 @@ class chain_plugin_impl {
    bool                             accept_transactions = false;
    bool                             api_accept_transactions = true;
```
Comment (larryk85) [inline plugins/chain_plugin/chain_plugin.cpp:150]: formatting (lines 151-154, 156, 158)

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 23]
```diff
@@ -110,24 +109,24 @@ class read_only {
...
+      std::optional<uint32_t>  upper_bound;
       uint32_t            limit = 10;
       bool                search_by_block_num = false;
       bool                reverse = false;
    };
 
    struct get_activated_protocol_features_results {
       fc::variants        activated_protocol_features;
-      optional<uint32_t>  more;
+      std::optional<uint32_t>  more;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:129]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 24]
```diff
@@ -162,7 +161,7 @@ class read_only {
       fc::time_point             last_code_update;
       fc::time_point             created;
 
-      optional<asset>            core_liquid_balance;
+      std::optional<asset>            core_liquid_balance;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:164]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 25]
```diff
@@ -193,7 +192,7 @@ class read_only {
       string                 wast;
       string                 wasm;
       fc::sha256             code_hash;
-      optional<abi_def>      abi;
+      std::optional<abi_def>      abi;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:195]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 26]
```diff
@@ -212,7 +211,7 @@ class read_only {
 
    struct get_abi_results {
       name                   account_name;
-      optional<abi_def>      abi;
+      std::optional<abi_def>      abi;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:214]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 27]
```diff
@@ -231,14 +230,14 @@ class read_only {
 
    struct get_raw_abi_params {
       name                   account_name;
-      optional<fc::sha256>   abi_hash;
+      std::optional<fc::sha256>   abi_hash;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:233]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 28]
```diff
@@ -231,14 +230,14 @@ class read_only {
...
    };
 
    struct get_raw_abi_results {
       name                   account_name;
       fc::sha256             code_hash;
       fc::sha256             abi_hash;
-      optional<chain::blob>  abi;
+      std::optional<chain::blob>  abi;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:240]: formatting

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 29]
```diff
@@ -356,7 +355,7 @@ class read_only {
    struct get_currency_balance_params {
       name             code;
       name             account;
-      optional<string> symbol;
+      std::optional<string> symbol;
```
Comment (larryk85) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:358]: formatting

File: plugins/history_plugin/include/eosio/history_plugin/history_plugin.hpp [hunk 30]
```diff
@@ -43,7 +42,7 @@ class read_only {
       struct get_actions_result {
          vector<ordered_action_result> actions;
          uint32_t                      last_irreversible_block;
-         optional<bool>                time_limit_exceeded_error;
+         std::optional<bool>                time_limit_exceeded_error;
```
Comment (larryk85) [inline plugins/history_plugin/include/eosio/history_plugin/history_plugin.hpp:45]: formatting

File: plugins/history_plugin/include/eosio/history_plugin/history_plugin.hpp [hunk 31]
```diff
@@ -52,7 +51,7 @@ class read_only {
 
       struct get_transaction_params {
          string                        id;
-         optional<uint32_t>            block_num_hint;
+         std::optional<uint32_t>            block_num_hint;
```
Comment (larryk85) [inline plugins/history_plugin/include/eosio/history_plugin/history_plugin.hpp:54]: formatting

File: plugins/http_plugin/http_plugin.cpp [hunk 32]
```diff
@@ -183,7 +183,7 @@ namespace eosio {
       public:
          // key -> priority, url_handler
          map<string,detail::internal_url_handler>  url_handlers;
-         optional<tcp::endpoint>  listen_endpoint;
+         std::optional<tcp::endpoint>  listen_endpoint;
```
Comment (larryk85) [inline plugins/http_plugin/http_plugin.cpp:186]: formatting

File: plugins/http_plugin/http_plugin.cpp [hunk 33]
```diff
@@ -193,20 +193,20 @@ namespace eosio {
          websocket_server_type    server;
 
          uint16_t                                    thread_pool_size = 2;
-         optional<eosio::chain::named_thread_pool>   thread_pool;
+         std::optional<eosio::chain::named_thread_pool>   thread_pool;
```
Comment (larryk85) [inline plugins/http_plugin/http_plugin.cpp:196]: formatting

File: plugins/http_plugin/http_plugin.cpp [hunk 34]
```diff
@@ -193,20 +193,20 @@ namespace eosio {
...
          std::atomic<size_t>                         bytes_in_flight{0};
          size_t                                      max_bytes_in_flight = 0;
          fc::microseconds                            max_response_time{30*1000};
 
-         optional<tcp::endpoint>  https_listen_endpoint;
+         std::optional<tcp::endpoint>  https_listen_endpoint;
```
Comment (larryk85) [inline plugins/http_plugin/http_plugin.cpp:201]: formatting

File: plugins/net_plugin/include/eosio/net_plugin/net_plugin.hpp [hunk 35]
```diff
@@ -29,7 +29,7 @@ namespace eosio {
 
         string                       connect( const string& endpoint );
         string                       disconnect( const string& endpoint );
-        optional<connection_status>  status( const string& endpoint )const;
+        std::optional<connection_status>  status( const string& endpoint )const;
```
Comment (larryk85) [inline plugins/net_plugin/include/eosio/net_plugin/net_plugin.hpp:32]: formatting

File: plugins/net_plugin/net_plugin.cpp [hunk 36]
```diff
@@ -270,7 +270,7 @@ namespace eosio {
       compat::channels::transaction_ack::channel_type::handle  incoming_transaction_ack_subscription;
 
       uint16_t                                  thread_pool_size = 2;
-      optional<eosio::chain::named_thread_pool> thread_pool;
+      std::optional<eosio::chain::named_thread_pool> thread_pool;
```
Comment (larryk85) [inline plugins/net_plugin/net_plugin.cpp:273]: formatting

File: plugins/net_plugin/net_plugin.cpp [hunk 37]
```diff
@@ -588,7 +588,7 @@ namespace eosio {
       std::atomic<go_away_reason>           no_retry{no_reason};
 
       mutable std::mutex          conn_mtx; //< mtx for last_req .. local_endpoint_port
-      optional<request_message>   last_req;
+      std::optional<request_message>   last_req;
```
Comment (larryk85) [inline plugins/net_plugin/net_plugin.cpp:591]: formatting

File: plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp [hunk 38]
```diff
@@ -57,15 +57,15 @@ class producer_plugin : public appbase::plugin<producer_plugin> {
    };
 
    struct get_account_ram_corrections_params {
-      optional<account_name>  lower_bound;
-      optional<account_name>  upper_bound;
+      std::optional<account_name>  lower_bound;
```
Comment (larryk85) [inline plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp:60]: formatting

File: plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp [hunk 39]
```diff
@@ -57,15 +57,15 @@ class producer_plugin : public appbase::plugin<producer_plugin> {
...
+      std::optional<account_name>  upper_bound;
       uint32_t                limit = 10;
       bool                    reverse = false;
    };
 
    struct get_account_ram_corrections_result {
       std::vector<fc::variant> rows;
-      optional<account_name>   more;
+      std::optional<account_name>   more;
```
Comment (larryk85) [inline plugins/producer_plugin/include/eosio/producer_plugin/producer_plugin.hpp:68]: formatting

File: plugins/state_history_plugin/state_history_plugin.cpp [hunk 40]
```diff
@@ -42,12 +42,12 @@ auto catch_and_log(F f) {
 
 struct state_history_plugin_impl : std::enable_shared_from_this<state_history_plugin_impl> {
    chain_plugin*                                              chain_plug = nullptr;
-   fc::optional<state_history_traces_log>                     trace_log;
-   fc::optional<state_history_chain_state_log>                chain_state_log;
+   std::optional<state_history_traces_log>                     trace_log;
```
Comment (larryk85) [inline plugins/state_history_plugin/state_history_plugin.cpp:45]: formatting

File: plugins/state_history_plugin/state_history_plugin.cpp [hunk 41]
```diff
@@ -78,7 +78,7 @@ struct state_history_plugin_impl : std::enable_shared_from_this<state_history_pl
       bool                                       sending  = false;
       bool                                       sent_abi = false;
       std::vector<std::vector<char>>             send_queue;
-      fc::optional<get_blocks_request_v0>        current_request;
+      std::optional<get_blocks_request_v0>        current_request;
```
Comment (larryk85) [inline plugins/state_history_plugin/state_history_plugin.cpp:81]: formatting

File: plugins/trace_api_plugin/include/eosio/trace_api/chain_extraction.hpp [hunk 42]
```diff
@@ -120,7 +120,7 @@ class chain_extraction_impl_type {
    StoreProvider                                                store;
    exception_handler                                            except_handler;
    std::map<transaction_id_type, cache_trace>                   cached_traces;
-   fc::optional<cache_trace>                                    onblock_trace;
+   std::optional<cache_trace>                                    onblock_trace;
```
Comment (larryk85) [inline plugins/trace_api_plugin/include/eosio/trace_api/chain_extraction.hpp:123]: formatting

File: plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp [hunk 43]
```diff
@@ -97,7 +97,7 @@ struct txn_test_gen_plugin_impl {
    uint64_t _txcount = 0;
 
    uint16_t                                             thread_pool_size;
-   fc::optional<eosio::chain::named_thread_pool>        thread_pool;
+   std::optional<eosio::chain::named_thread_pool>        thread_pool;
```
Comment (larryk85) [inline plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp:100]: formatting

File: programs/eosio-tester/main.cpp [hunk 44]
```diff
@@ -183,10 +183,10 @@ struct test_chain {
    fc::temp_directory                                dir;
    std::unique_ptr<eosio::chain::controller::config> cfg;
    std::unique_ptr<eosio::chain::controller>         control;
-   fc::optional<scoped_connection>                   applied_transaction_connection;
-   fc::optional<scoped_connection>                   accepted_block_connection;
+   std::optional<scoped_connection>                   applied_transaction_connection;
```
Comment (larryk85) [inline programs/eosio-tester/main.cpp:186]: formatting

File: unittests/whitelist_blacklist_tests.cpp [hunk 45]
```diff
@@ -76,7 +76,7 @@ class whitelist_blacklist_tester {
    private:
       fc::temp_directory                tempdir; // Must come before chain
    public:
-      fc::optional<Tester>              chain;
+      std::optional<Tester>              chain;
```
Comment (larryk85) [inline unittests/whitelist_blacklist_tests.cpp:79]: formatting

**Comment by TimothyBanks** [comment]
@larryk85 Updated

**Comment by heifner** [comment]
Is the abieos changes expected? Can't tell because of the force-push.

**Comment by TimothyBanks** [comment]
Replay completed successfully.
https://buildkite.com/EOSIO/auto-eosio-full-replay-test/builds/245#_
develop has been merged. 

Please take another look.

**Comment by TimothyBanks** [comment]
@larryk85 @heifner @allenhan2 I think this is ready to go.  Please take another look.

**Pull request merged**